src/DataModel/IMetadataDescription.ts

```diff
@@ -0,0 +1,11 @@
+import {IMetadata} from "./IMetadata";
+import {IClass} from "./IClass";
+import {IOperation} from "./IOperation";
+import HydraClient from "../HydraClient";
+
+export interface IMetadataDescription extends IMetadata
```

Comment:
Similarly to the above. Hydra specification doesn't speak about any metadata anywhere. The properties below suggest to me that maybe this would actually be the `IApiDocumentation` interface? And the current `IApiDocumentation` could become simply `IApi`?

I'm sorry I contradict earlier comments but I'm coming up with this stuff as I go through and these are my thoughts :)